wlm/src/test/java/ai/djl/serving/wlm/ModelInfoTest.java

```diff
@@ -244,4 +247,75 @@ public void testInitModel() throws IOException, ModelException {
         model.initialize();
         assertEquals(model.getEngineName(), "PyTorch");
     }
+
+    @Test
+    public void testInferLMIEngine() throws IOException, ModelException {
+        Path modelStore = Paths.get("build/models");
+        Path modelDir = modelStore.resolve("lmi_test_model");
+        Files.createDirectories(modelDir);
+        Path deepspeedLocation = Paths.get("/usr/local/bin/deepspeed");
+        boolean deepspeedExisted = true;
+        if (!Files.exists(deepspeedLocation)) {
+            Files.createDirectories(deepspeedLocation);
+            deepspeedExisted = false;
+        }
+        Path fastertransformerLocation = Paths.get("/usr/local/backends/fastertransformer");
+        boolean fastertransformerExisted = true;
+        if (!Files.exists(fastertransformerLocation)) {
+            Files.createDirectories(fastertransformerLocation);
+            fastertransformerExisted = false;
+        }
```

Comment:
I added this for the unit tests, but not a big fan of it. We can instead test this with some integration tests?